demo.cast

```diff
@@ -0,0 +1,1559 @@
+{"version": 2, "width": 117, "height": 70, "timestamp": 1614956408, "env": {"SHELL": "/bin/zsh", "TERM": "xterm-256color"}}
```

Comment:
I guess you didn't want to commit this file?

Reply:
I'm not sure, that was there in @xcoulon original commit.

Reply:
Indeed, it was not intended. Although, one may consider it an historical piece of art!
Joke aside, I will remove it

Reply:
removed in b7a9f4f